### PR TITLE
feature: add helm chart; take state from last working milestone

### PR DESCRIPTION
**Pull request by sjentzsch**
Adding a helm-chart for checkmk kubernetes agent.

### CODE REVIEW

File: deploy/charts/checkmk/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+### Multiple releases
+
+The same chart can be used to run multiple checkmk instances in the same cluster (or even the same namespace) if required. To achieve this, just rename the `RELEASE_NAME` when installing.
+
+## Debug
+
+To debug in-cluster, you can launch a debug pod with network tools via
+
+```console
+kubectl -n [RELEASE_NAMESPACE] run -it debug --rm --image praqma/network-multitool --restart=Never -- sh
```
Comment (okin) [inline deploy/charts/checkmk/README.md:84]: This has been renamed to `wbitt/network-multitool` (as per [this change](https://github.com/Praqma/Network-MultiTool/commit/c3d4e04c353cdbf8dcbcb8f57190056a66c273c0)).

Comment (sjentzsch) [reply deploy/charts/checkmk/README.md:84]: Thanks! Good to know. Will update.

**Comment by lisa-pi** [comment]
Hi Sören, thanks a lot for your work! We greatly appreciate it. We have tested the charts and can confirm that they work for us. Please squash your commits. We will then be able to merge your change.

**Comment by lisa-pi** [comment]
Hi Sören, your PR has been merged with commit I3d12bf20fee08ed7950cbc65c596913b903c5afc
Thanks! :)

**Pull request closed without merging**